up42/auth.py

```diff
@@ -42,9 +43,9 @@ def is_http_429_error(exception):
 class Auth:
     def __init__(
         self,
-        cfg_file: Union[str, Path, None] = None,
```

Comment:
Is this change in order absolutely necessary? I think this could break users workflows in case they make use of order of arguments instead of keyword arguments.

Reply:
Reverted. Think it would be good to have the config file as the second option, but true, maybe better suited when going to 1.0.0.